### ISSUE TITLE
Pickup as shipping option

### ISSUE THREAD

**Issue by jstrnbrg** [shiftdevices/saleor#22]
We want to offer our customers the option to order anonymously and pick the order up at one of our partners. 

- [ ] on /checkout/shipping-address/ implement option to choose pickup, when selected it should only ask for an email address and show a dropdown to select the pickup location. Price for pickup should be specify-able in the standard shipping method settings. 

- [ ] /checkout/shipping-method/ should be skipped in this case

- [ ] /checkout/summary/ should not show the Billing address component and just summarise the order + notes field + "order & pay" field. 

Then the user should be forwarded to the normal payment page 

- [ ] Possible Pickup locations should be specified in the dashboard. Pickup Location model should have these fields: Name, Address, Email, Phone Number

**Comment by elvissio** [comment]
In the payment page we should limit options to crypto (bitcoin?) only.

**Comment by jstrnbrg** [comment]
I don't think so. Someone just might prefer to pick it up for convenience reasons and not for privacy concerns. i.e. you live in a swiss mountain village with a kiosk and you want your bitbox today. 
In that case they might want to pay with card as their reason to use the pick up service is the speed of delivery compared to anonymity. 

I'm assuming each kiosk would have a few bitboxes in stock compared to shipping them there on demand. Seems logistically more efficient and reduces the workload on our partner. @elvissio

**Comment by elvissio** [comment]
That opens a completely new level of cooperating w Valora (KKiosk). I'd leave more options open for the future, when we see how the initial, anonymous, pickup story goes. 
Also, do we want KKiost is selling premium cybersecurity device? What message do we send to the market w that. I'd leave those options open for next possible steps.

Also:
"I'm assuming each kiosk would have a few bitboxes in stock compared to shipping them there on demand. Seems logistically more efficient and reduces the workload on our partner."
It is logistically more efficient for us, and increases the workload for our partner! They might not want doing it in 1st place at all. That's also a reason I'd avoid that for the initial setup. 

Once I will explain you my proven "closed umbrella" approach. Maybe at a beer ;)

**Comment by jstrnbrg** [comment]
ok gerne ;)
